pom.xml

```diff
@@ -4,26 +4,137 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.demo</groupId>
-    <artifactId>java-school</artifactId>
+    <groupId>com.java-school</groupId>
```

Comment:
could you rename it without dash like >com.javaschool or com.tsystems.javaschool.